### PR TITLE
[#131] kube2iam implementation

### LINKED ISSUE #131: Investigate iam2kube for legion purposes
We should be able to grant AWS resources access for Legion Pods and Containers. Worth testing iam2kube  with S3 access to Enclave pods.

### PR DESCRIPTION
**Pull request by alexander-semenets**
This PR fixes #131 issue
ok to test

### CODE REVIEW

**Comment by alexander-semenets** [comment]
test this please

**Comment by alexander-semenets** [comment]
test this please

**Comment by alexander-semenets** [comment]
test this please

File: deploy/ansible/roles/legion_enclave/templates/airflow_s3_access_policy.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+     {
+      "Effect": "Allow",
+      "Action": ["s3:*"],
+      "Resource": [
+        "arn:aws:s3:::{{ airflow_s3_bucket_name }}/{{ enclave }}",
```
Comment (akharlamov) [inline deploy/ansible/roles/legion_enclave/templates/airflow_s3_access_policy.yaml.j2]: Is it airflow log bucket or data bucket ?

Comment (aliaksandr-d) [reply deploy/ansible/roles/legion_enclave/templates/airflow_s3_access_policy.yaml.j2]: it's a path that this Airflow can access. It's used for data and logs

**Comment by alexander-semenets** [comment]
test this please

**Comment by alexander-semenets** [comment]
test this please

**Comment by akharlamov** [comment]
@alexander-semenets is it ready for review?

**Comment by alexander-semenets** [comment]
@akharlamov yes please, the CI passed successfully so we are good to go (the last failure in the list is because of #89 issue on CI side)

**Pull request merged**